sdk/clientcore/core/src/main/java/module-info.java

```diff
@@ -34,7 +35,6 @@
     exports io.clientcore.core.traits;
     exports io.clientcore.core.utils;
     exports io.clientcore.core.utils.configuration;
-    exports io.clientcore.core.implementation.http.client;
     exports io.clientcore.core.implementation.http;
```

Comment:
Can we remove this implementation package too?

Reply:
To do so it requires complex code changes that I want to make its own PR so we can review them more appropriately.